tests/build_with_options.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+
+echo $PWD
+# Protect against mistakes by not just blindly deleting the second input
+# argument.
+rm -Rf "${2}_prev"
+mv $2 "${2}_prev"
+mkdir -p "$2"
+cd "$2"
+cmake $3 $1
+make -j5 aktualizr
```

Comment:
Why is this needed?  There shouldn't be any case where a clean build gives different results to an incremental build, and if there is, we should fix _that_ bug.  I'm worried that starting from scratch here will slow down the development build process.